closed/custom/common/JavaCompilation.gmk

```diff
@@ -18,5 +18,5 @@
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
 # ===========================================================================
 
-# OpenJ9 requires more lenient javac warnings
-JAVAC_WARNINGS := -Xlint:-removal
+# OpenJ9 requires allowing warnings
+JAVA_WARNINGS_ARE_ERRORS :=
```

Comment:
Perhaps it would be better to put this (temporarily) in `custom-spec.gmk.in` until we can fix those warnings, then there would be no need to modify `make/common/JavaCompilation.gmk`.

Reply:
It doesn't work. I tried https://github.com/ibmruntimes/openj9-openjdk-jdk/compare/openj9-staging...pshipton:werror2?expand=1

Reply:
I hadn't reconfigured, it's working after doing so. I'll update the PR.